guides/micronaut-cloud-database-base/metadata.json

```diff
@@ -1,6 +1,5 @@
 {
   "publish": false,
-  "authors": ["Burt Beckwith"],
```

Comment:
I think this was fine as it was: you authored the base guide, then I feel appropriate to have you as co-author of all guides that use such base.